sdk/formrecognizer/Azure.AI.FormRecognizer/src/autorest.md

```diff
@@ -6,8 +6,9 @@ Run `dotnet build /t:GenerateCode` to generate code.
 > see https://aka.ms/autorest
 
 ``` yaml
-input-file:
-    - https://github.com/Azure/azure-rest-api-specs/blob/788507c386197b1ba7878fa00fe30871b8b01f22/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json
+tag: release_2_1_preview.2
```

Comment:
also curious, is there a place where these parameters are defined?

Reply:
This tag is defined [here](https://github.com/Azure/azure-rest-api-specs/blob/788507c386197b1ba7878fa00fe30871b8b01f22/specification/cognitiveservices/data-plane/FormRecognizer/readme.md#release-21-preview2).

Reply:
The readme.md file maps each valid tag to an input file ([v2.1-preview.2 specified here](https://github.com/Azure/azure-rest-api-specs/blob/788507c386197b1ba7878fa00fe30871b8b01f22/specification/cognitiveservices/data-plane/FormRecognizer/readme.md#release-21-preview2)).

Reply:
Missed Shivangi's comments. At least our answers match.